README.md

```diff
@@ -57,7 +57,8 @@ By default, Kuzu Explorer is launched in read-write mode, which means that you c
 
 ```bash
 docker run -p 8000:8000 \
-           -v /absolute/path/to/database:/database \
+           -v {path to the directory containing the database file}:/database \
```

Comment:
I like the current style of listing an actual path example tbh.

Reply:
I think `path/to/the/directory/containing/the/database/file` is too long and not readable

Reply:
Since now we implemented single-file database I think database directory is ambiguous and we need to be explicit here.

Reply:
I'll leave it your best judgement.

The reason I like the path-based example is that it visually shows what the actual path looks like. Maybe `/directory/for/db/file` or something.

It makes editing the path easier. I can copy paste the example into a terminal and use a single `alt-w` to clear out the example to paste in an actual path. Now I've to use multiple deletes as there are spaces in the example. Maybe use `-` at the very least.